x-pack/solutions/security/plugins/elastic_assistant/server/ai_assistant_data_clients/conversations/field_maps_configuration.ts

```diff
@@ -37,11 +37,6 @@ export const conversationsFieldMap: FieldMap = {
     array: false,
     required: true,
   },
-  is_default: {
-    type: 'boolean',
-    array: false,
-    required: false,
-  },
```

Comment:
Side note: it would be nice if we kept a version history of mappings changes so we could easily visualize what changed and when similar to what they did with the [alerts index](https://github.com/elastic/kibana/blob/da25d13a2ac5be97216e202bf5d6bda6ff920b3b/x-pack/solutions/security/plugins/security_solution/common/api/detection_engine/model/alerts/README.md). I don't think we need to go the whole versioned schemas route like they did, but a simple changelog logging mappings changes w/ reason could be helpful vs the git log of this file.